addons/web/static/tests/views/calendar_tests.js

```diff
@@ -1599,6 +1600,64 @@ QUnit.module('Views', {
             "should be one event in the all day row");
         calendar.destroy();
     });
+
+    QUnit.test('test if the CalendarView create memory leak', function (assert) {
+        assert.expect(2);
+
+        var activeWidgets = [];
+        var initWidget = mixins.ParentedMixin.init;
+        mixins.ParentedMixin.init = function () {
+            activeWidgets.push(this);
```

Comment:
increment a counter instead (named nbInstances for example)

Reply:
It's easier to fix with the widgets

Reply:
don't think so